examples/low_level.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+from envision.client import Client as Envision
+from examples import default_argument_parser
+from smarts.core.agent import AgentSpec
+from smarts.core.agent_interface import AgentInterface, AgentType
+from smarts.core.scenario import Scenario
+from smarts.core.smarts import SMARTS
+from smarts.core.sumo_traffic_simulation import SumoTrafficSimulation
+
+logging.basicConfig(level=logging.INFO)
+
+
+def main(scenarios, headless, seed):
+    agent_spec = AgentSpec(
+        interface=AgentInterface.from_type(AgentType.Laner, max_episode_steps=None),
+        agent_builder=None,
+        observation_adapter=None,
+    )
+
+    smarts = SMARTS(
+        agent_interfaces={},
+        traffic_sim=SumoTrafficSimulation(headless=True, auto_start=True),
+    )
+    smarts2 = SMARTS(
+        agent_interfaces={},
+        traffic_sim=SumoTrafficSimulation(headless=True, auto_start=True),
+    )
+
+    scenarios_iterator = Scenario.scenario_variations(
+        scenarios,
+        list([]),
+    )
+
+    smarts.reset(next(scenarios_iterator))
+
+    for _ in range(80):
+        smarts.step({})
+        smarts.attach_sensors_to_vehicles(
+            agent_spec, smarts.vehicle_index.social_vehicle_ids()
+        )
```

Comment:
When a social vehicle enters a bubble, sensors are also added to the vehicle, which causes this test to be non-deterministic -- it will sometimes assert from trying to add a sensor twice because of that.

Reply:
That problem seems to relate to https://github.com/huawei-noah/SMARTS/issues/411